packages/dds/legacy-dds/src/test/sharedArray.spec.ts

```diff
@@ -0,0 +1,204 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "node:assert";
```

Comment:
i think it would be good to put the test files in a folder as well.

Reply:
you mean inside array folder?

Reply:
`test/array/<the files`